README.md

```diff
@@ -6,8 +6,8 @@
 
 ----
 
-![Build](https://github.com/pymc-labs/pymc-marketing/actions/workflows/ci.yml/badge.svg)
 ![Test](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test.yml/badge.svg)
+![Test Notebook](https://github.com/pymc-labs/pymc-marketing/actions/workflows/test_notebook.yml/badge.svg)
```

Comment:
This is just duplicate now with the next line

Reply:
ahhh

Reply:
fixed!